I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 122, in script
    jump scene_2_1_1
ScriptError: could not find label 'scene_2_1_1'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 122, in script
    jump scene_2_1_1
  File "C:\Users\joann\Downloads\renpy-8.0.3-sdk\renpy\ast.py", line 1974, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\joann\Downloads\renpy-8.0.3-sdk\renpy\script.py", line 927, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'scene_2_1_1'.

Windows-10-10.0.22000 AMD64
Ren'Py 8.0.3.22090809
BetaGame 1.0
Wed Sep 28 01:41:31 2022
